assets/css/styles.css

```diff
@@ -0,0 +1,631 @@
+
+              .user-name {
+                font-weight: 600;
+              }
+            }
+          }
+
+          .user-review {
+            line-height: 28px;
+          }
+        }
+      }
+    }
+  }
+
+  .blog-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 80px;
+
+    .blog-section__list {
+      display: flex;
+      justify-content: center;
+      gap: 32px;
+
+      .blog-card {
+        display: flex;
+        flex-direction: column;
+        border-radius: 0 96px 0 40px;
+        border: 1px solid var(--neutral-grey);
+        padding: 16px;
+        gap: 16px;
+        overflow: hidden;
+
+        .blog-card__img {
+          max-width: 373px;
+          height: 320px;
+          object-fit: cover;
+          border-radius: 0 80px;
+        }
+
+        .blog-card__user-date {
+          display: flex;
+          justify-content: space-between;
+          font-size: 14px;
+
+          img {
+            width: 17px;
+            height: 17px;
+          }
+        }
+
+        .blog-card__description {
+          font-size: 24px;
+          font-weight: 600;
+          line-height: 33.6px;
+        }
+
+        .blog-card__view-details {
+          display: flex;
+          justify-content: end;
+          align-items: center;
+          gap: 8px;
+          padding: 0 16px;
+
+          span {
+            font-size: 16px;
+          }
+        }
+      }
+    }
+  }
+
+  .footer-section {
+    display: flex;
+    gap: 80px;
+    color: #fff;
+    background-color: var(--neutral-black);
+
+    .footer-section__left {
+      display: flex;
+      flex-direction: column;
+      gap: 32px;
+
+      .logo {
+        height: 58px;
+        width: fit-content;
+      }
+
+      .description {
+        line-height: 32px;
+      }
+
+      .contact {
+        line-height: 30px;
+      }
+    }
+
+    .footer-section__right {
+      display: flex;
+      flex-direction: column;
+      gap: 40px;
+
+      .footer-section__links {
+        display: flex;
+        justify-content: space-between;
+
+        .footer-section__link-group {
+          display: flex;
+          flex-direction: column;
+          gap: 24px;
+
+          .link-group__name {
+            font-size: 28px;
+            font-family: "Playfair Display";
+            text-transform: capitalize;
+          }
+
+          ul {
+            display: flex;
+            flex-direction: column;
+            gap: 16px;
+
+            li {
+              font-size: 16px;
+              text-transform: capitalize;
+            }
+
+            &.social-media {
+              display: flex;
+              flex-direction: row;
+              gap: 8px;
+              opacity: 70%;
+
+              li {
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                width: 48px;
+                height: 48px;
+                background: #ffffff14;
```

Comment:
What is this hex code?

Reply:
Here is the hex code I received when copying, maybe it's a white color with an opacity of 8%. Do I need to rewrite it?
![image](https://github.com/user-attachments/assets/1123c1d0-d170-416f-a62e-dca290eb223d)